config/application.rb

```diff
@@ -25,5 +25,6 @@ class Application < Rails::Application
     config.active_storage.variant_processor = :mini_magick
     config.active_storage.content_types_to_serve_as_binary.delete("image/svg+xml")
     config.serve_static_assets = true
+    config.time_zone = ActiveSupport::TimeZone[Time.now.strftime("%z").delete("0").to_i]
```

Comment:
Given that it will be running on heroku servers which are themselves in UTC, I do not expect this to solve our problem. But also, let's not override server timezone.